padre_meddea/io/file_tools.py

```diff
@@ -46,8 +46,8 @@ def read_file(filename: Path):
     Examples
     --------
     """
-    if filename.suffix == "bin":  # raw binary file
-        result = read_raw_file(filename)
+    if filename.suffix == ".bin":  # raw binary file
+        result = read_raw_file(Path(filename))
     elif filename.suffix == "fits":  # level 0 or above
```

Comment:
Should there be a "." in front of "fits"?

Reply:
Good catch. I actually fixed this in my other pull request.